overlay/files-alpine/etc/init.d/kairos-agent

```diff
@@ -2,15 +2,11 @@
 
 depend() {
   provide kairos-agent
+  after cos-setup-network
+  use net
 }
 
-supervisor=supervise-daemon
-name="kairos-agent"
-command="kairos-agent start"
-command_args="--restart"
```

Comment:
Found out that leaving like that it would exit as crashed all the time.

Openrc is a bit weird, I could only get it to work as a one shot script that exits correctly like this